WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/HelpAndSupportViewController.swift

```diff
@@ -147,15 +147,17 @@ private extension HelpAndSupportViewController {
                     .contactSupport,
                     .myTickets,
                     .contactEmail,
-                    .applicationLog]
+                    .applicationLog,
+                    .systemStatusReport]
         }
 
         return [.helpCenter,
                 .contactSupport,
                 .contactWCPaySupport,
                 .myTickets,
                 .contactEmail,
-                .applicationLog]
+                .applicationLog,
+                .systemStatusReport]
```

Comment:
nit: might we want to show the SSR row behind a feature flag, if there is still some work left and the feature might be wrapped up after release 8.3?

Reply:
Good thought! I mentioned above that I thought I could finish all this within this week 😅. But it's neater to add the flag just in case, so I updated that in 49aae11f2f5fb455ef9ce98e1d6f9165d2f56bdc.